### ISSUE TITLE
Wav2Vec 2: How to control batch size?

### ISSUE THREAD

**Issue by weiyengs** [pytorch/fairseq#3262]
Hi there,
I can't seem to find the hyper-parameter to control the batch size. I'm currently using the command like this:

python train.py 'examples/wav2vec/train_data' --save-dir 'examples/wav2vec/ftune_models' \
--wer-args '("examples/wav2vec/train_data/lm.bin","examples/wav2vec/train_data/lexicon.txt",2,-1)' \
--w2v-path examples/wav2vec/ptrain_models/wav2vec_small.pt --cpu --device-id 0 \
--post-process letter --valid-subset train --no-epoch-checkpoints --best-checkpoint-metric wer --num-workers 4 \
--max-update 100 --sentence-avg --task audio_pretraining --arch wav2vec_ctc  \
--labels ltr --apply-mask --mask-selection static --mask-other 0 --mask-length 10 --mask-prob 0.5 --layerdrop 0.1 \
--mask-channel-selection static --mask-channel-other 0 --mask-channel-length 64 --mask-channel-prob 0.5 --zero-infinity \
--feature-grad-mult 0.0 --freeze-finetune-updates 10000 --validate-after-updates 10000 --optimizer adam \
--adam-betas '(0.9, 0.98)' --adam-eps 1e-08 --lr 2e-05 --lr-scheduler tri_stage --warmup-steps 8000 --hold-steps 32000 \
--decay-steps 40000 --final-lr-scale 0.05 --final-dropout 0.0 --dropout 0.0 --activation-dropout 0.1 --criterion ctc \
--attention-dropout 0.0 --max-tokens 1280000 --seed 2337 --log-format json --log-interval 10 --ddp-backend no_c10d

Thanks!

**Comment by lematt1991** [comment]
You should use the [min_sample_size/max_sample_size](https://github.com/pytorch/fairseq/blob/master/fairseq/tasks/audio_pretraining.py#L56-L61) arguments
